### DISCUSSION TITLE
Is it possible to bookmark issues so you don't forget me as a reminder?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31058] [Projects and Issues] [answered]
Hi everyone! Is it possible to mark issues so you don't forget me as a reminder?

#### Something like this?
![image](https://user-images.githubusercontent.com/111701513/187046385-60915ac0-2a31-46f9-8bdf-60ac84626344.png)

**Comment by jfullstackdev** [comment, marked as answer]
you can subscribe to receive notifs 

or you can select it at the watch repo tab, custom only for issues

![image](https://user-images.githubusercontent.com/47092464/187081274-b5f0ad7b-c4c0-4fef-b9f9-904f54a34b92.png)

as for bookmarking, just the regular Bookmark feature of a web browser

**Reply by ghost** [reply]
Hey there! Whats up?! Currently, it is possible to create a list of favorite repositories. I wanted to create a list of favorite issues.

**Reply by jfullstackdev** [reply]
as of now, there isn't a direct way to do that, 

by including the repos with your favorite issues, that I think will still do the job

or manually save the URLs of each favorite issue and, say, create your own bookmarks on a README file

**Reply by VladimirFokow** [reply]
the disadvantage of bookmarking in the web browser: isn't synced across all devices

**Comment by batmandarin** [comment]
#[@courtneycl ](url)

**Comment by dserodio** [comment]
GitHub should support this natively, but meanwhile, [Octotree](https://www.octotree.io/) is the workaround I currently use

**Reply by ghost** [reply]
Hi dserodio.

> GitHub should support this natively, but meanwhile, [Octotree](https://www.octotree.io/) is the workaround I currently use

thank you for feedback here.

**Comment by qwadrox** [comment]
For this specific reason, I created a Chrome extension: https://chromewebstore.google.com/detail/github-issue-glance/kallgfejdofmpelngmpklcddomojplod?authuser=0&hl=en
It allows you to bookmark and manage bookmarked issues and that's it.
